### PR TITLE
Update readme for radio fragment

### PR DESCRIPTION
**Pull request by bseeger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +16,9 @@ The library includes tooling for:
 - Input Validations
     - Using [JPA Validation](https://www.baeldung.com/spring-boot-bean-validation)
 - Address Validation using [Smarty](https://smarty.com/)
-- A set of [Thymeleaf fragments](https://github.com/codeforamerica/form-flow/tree/main/src/main/resources/templates/fragments) that create a library of reusable HTML components for Inputs, Screens, Forms, etc.
+- A set
```
Comment (bseeger) [inline README.md:19]: So much of this is just intellij styling this.  Not sure why it wasn't already styled correctly.

File: README.md [hunk 2]
```diff
@@ -453,50 +462,52 @@ An example `Submission` object can be seen below:
...
-  private Map<String, Object> inputData;
+    @Type(JsonType.class)
+    @Column(name = "input_data", columnDefinition = "jsonb")
+    private Map<String, Object> inputData;
 
-  @Type(JsonType.class)
-  @Column(name = "url_params", columnDefinition = "jsonb")
-  private Map<String, String> urlParams;
+    @Type(JsonType.class)
+    @Column(name = "url_params", columnDefinition = "jsonb")
```
Comment (bseeger) [inline README.md:477]: We did change from 2 - 4 spaces, so I'm guessing that's why this changed here.

File: README.md [hunk 3]
```diff
@@ -1232,19 +1246,19 @@ checkbox option fragment
...
+in test it would be `gender[]`. This is done so that we can differentiate between single value
+checkboxes
+and multi-value in test.
 
 #### Radio
 
 Radio inputs are used to gather a single selection from a set of options. They are used in unison
-with
-a `radioFieldset` fragment to create a group of radio inputs that are all related to each other.
+with a `radioFieldset` fragment to create a group of radio inputs that are all related to each
```
Comment (bseeger) [inline README.md:1257]: This is where my actual changes begin.

File: README.md [hunk 4]
```diff
@@ -1253,33 +1267,40 @@ An example of a radio input:
...
+fieldset's legend. `radio`, too, has an optional `radioHelpText` field which will appear
+under the label's description text.
 
 For convenience, we have provided a `cfa:inputFieldsetWithRadio` live template which can be used to
-quickly
-create groupings of radio inputs. Not that when using this template, you can copy the inner radio
-option fragment
-as many times as you like to create the necessary number of radio options.
+quickly create groupings of radio inputs. Not that when using this template, you can copy the inner
+radio option fragment as many times as you like to create the necessary number of radio options.
```
Comment (bseeger) [inline README.md:1303]: My actual changes end here.

**Review by spokenbird — APPROVED** [review]
Looks good to me!

**Pull request merged**